src/styles/components/_menu.scss

```diff
@@ -28,6 +28,7 @@
   background: map-get($roles, menu);
   height:100%;
   color: $charcoal;
+  overflow-y: auto;
```

Comment:
I considered that when I saw scrollable, but the specs don't indicate fading. If we decide we want fading, I love the idea of reuse.